tpls/desul/include/desul/atomics/Generic.hpp

```diff
@@ -16,10 +16,11 @@ SPDX-License-Identifier: (BSD-3-Clause)
 
 #include <type_traits>
 
+#include <desul/atomics/Fetch_Op.hpp>
+
 #include <desul/atomics/Generic_Device.hpp>
 #include <desul/atomics/Generic_Host.hpp>
 
-#include <desul/atomics/Fetch_Op.hpp>
```

Comment:
@crtrott this is actually awkward.  The latest version of the PR on the desul side had these include reordered with Clang-Format and the alphabetic order happens to do what we want...